packages/gitbook/src/app/(site)/fetch.ts

```diff
@@ -54,6 +64,19 @@ export async function fetchContentData() {
     };
 }
 
+function getSiteSectionsList(siteSectionId?: string, sections?: SiteSection[] | null) {
```

Comment:
This signature is a bit confusing. Why not `(sections: SiteSection[], siteSectionId: string)` and then do `const siteSections = content.siteSectionId && siteStructure.sections ? parseSiteSections(...) : null`

Reply:
Good point. I was thinking I'd encapsulate it in the function but it definitely makes the function signature confusing.